### ISSUE TITLE
how to make excludes from monitoring devices?

### ISSUE THREAD

**Issue by qnick** [calston/tensor#21]
I have many virtual network interfaces on my nodes, and tensor.sources.linux.basic.Network grab all of them and sends info into riemann, creating a lot of 'rubbish' traffic. Can I use smth like that?
```yaml
  - service: network
    source: tensor.sources.linux.basic.Network
    ifaces: eth0 # <-- monitor only this iface
    interval: 30.0
    ttl: 120
```
The same question about storage devices (DiskFree and DiskIO sources).

**Comment by calston** [comment]
Each source would have to implement that, I agree it's a good idea.

I'm wondering if the regex filter system couldn't also be used to filter out unwanted metrics from other things like SNMP and SFlow - but it's probably more efficient not to generate them at all.

Either way I'll have to add this as a feature

**Comment by calston** [comment]
Resolved in #22
